### PR TITLE
[MRG] Fix plot_properties.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/4185

### CODE REVIEW

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -761,7 +761,7 @@ def _plot_ica_topomap(ica, idx=0, ch_type=None, res=64, layout=None,
 
     if merge_grads:
         from ..channels.layout import _merge_grad_data
-        data = _merge_grad_data(data)
+        data = _merge_grad_data(data.ravel())
```
Comment (agramfort) [inline mne/viz/topomap.py]: _merge_grad_data docstring says it expects 

data : array, shape = (n_channels, n_times)

so I guess ravel() is bad or docstring is wrong

**Comment by jaeilepp** [comment]
Hmm. Yes, looks like the problem is actually in [line 759](https://github.com/mne-tools/mne-python/blob/master/mne/viz/topomap.py#L759) where the data is forced to 2d. I wonder why.

**Comment by agramfort** [comment]
+1 for MRG if travis is happy.

I think we should also back port this

**Comment by mmagnuski** [comment]
looks good, +1 too

**Comment by agramfort** [comment]
@jaeilepp can you back port too?

**Comment by jaeilepp** [comment]
Done.

**Pull request merged**